xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -4791,6 +4791,9 @@ TEST_P(ParameterizedFp8GemmRewriteTest, DoNotRewriteToF8OnPreAda) {
   if (HasFp8Support()) {
     GTEST_SKIP() << "Test requires a pre-Ada GPU or an AMD GPU prior to MI300.";
   }
+  if (!IsCuda()) {
+      GTEST_SKIP() << "Skip this rewrite pattern on ROCm";
+  }
```

Comment:
IIUC, we only skip this `Fp8CublasTestsBothLegacyAndLt` in MI100, it's no problem on MI300

https://github.com/ROCm/frameworks-internal/issues/8572 

so we need something like this [CudaHopperOrRocmMI300()](https://github.com/ROCm/xla/blob/5d21f8b7ef5c4b0d3fbdfe415ea3461f61980cd6/xla/service/gpu/tests/gemm_rewrite_test.cc#L79-L84)

Reply:
This is skipped on MI300 due to condition at line 4791. We are skipping this now for pre - Ada GPU prior to MI300 on ROCm 6.2. This is a temporary fix on QA branch until @wenchenvincent  has a reproducer against hipBlasLt.